prov/efa/src/efa_domain.c

```diff
@@ -278,6 +278,92 @@ void efa_atfork_callback()
 	abort();
 }
 
+/* @brief Check when fork is requested and abort in unsupported cases
```

Comment:
This comment only apply to linux, would you please move it to inside "#ifndef WIN32".

Reply:
done